readme.txt

```diff
@@ -3,10 +3,15 @@ Description:
 The analyzers can potentially support different languages and different policies, enabling this tool to provide wide and deep scans across a repository with very little effort from the end user 
 
 Installation: 
-    The user first needs to download this repository and install it somewhere in their home directory. 
+    The user first needs to download this repository. 
     Python3, pip, and java are required dependecies 
```

Comment:
To make it clear, you can add these pre-req under separate "Pre-requisites" section